components/input/hotkey.cpp

```diff
@@ -14,17 +14,17 @@ namespace Input
         alt = false;
     }
 
-    Hotkey::Hotkey(const char *name)
-    {
-        std::string sname = name;
+    Hotkey::Hotkey(const char *name) : Hotkey(std::string(name)) {}
```

Comment:
Probably yes but as I remember this ctor is used with Python and I don't if ctor(std::string) would be working properly.